src/utils/graph/graph.js

```diff
@@ -1,8 +1,8 @@
 /*
-* The Kedro Viz Graph Layout Engine
-*
-* Refer to LAYOUT_ENGINE.md for description of the approach.
-*/
+ * The Kedro Viz Graph Layout Engine
```

Comment:
Similar thing here, much appreciated if you can avoid committing whitespace changes in unrelated files (unless Prettier is doing this, we should look into this if so!)

Reply:
Ah no problem, looks like there are some Prettier version conflicts we'll need to investigate later.